mdfreader/mdf.py

```diff
@@ -152,8 +154,8 @@ def add_channel(self, dataGroup, channel_name, data, master_channel, \
         """
         if channel_name in self:
             if self[channel_name][dataField] is not None:
-                channel_name += '_' + str(dataGroup)
-                self[channel_name] = {}
+            channel_name += '_' + str(dataGroup)
```

Comment:
First, thanks for your pull request !
This section has changed due to introduction of noDataLoading argument.
But the indent is necessary there otherwise the if does not make sense